src/test/scala/nl.knaw.dans.easy.bagstore/component/BagStoresSpec.scala

```diff
@@ -99,6 +99,37 @@ class BagStoresSpec extends TestSupportFixture
     }
   }
 
+  it should "result in a failure when a get is done on a hidden file with forceInactive=false" in {
+    val output = testDir.resolve("pruned-output/hidden/test")
+    inside(bagStore1.add(testBagPrunedA)) { case Success(result) =>
+      bagStores.copyToDirectory(result, output, skipCompletion = true) shouldBe a[Success[_]]
+      pathsEqual(testBagPrunedA, output.resolve("a")) shouldBe true
+
+      // deactivate deposit
+      inside(bagStore1.deactivate(result)) { case Success(()) =>
```

Comment:
Again so far this looks like a preparation, use fail in case of a failure. Perhaps extract methods for the preparations to distinguish those actions clearly form the actual test.